[IA-3080] Usability testing feedback (top 2 nice-to-have points)

cstrandw opened the pull request:
- Increased the size of the RStudio logo in the `Select an application` modal after the `+Create` button and in the `Cloud Environment Details` modals
- Removed unnecessary arrow from the `Select an application` modal after clicking the `+Create` button
- Added new `r-bio-logo.svg` that has larger text to use instead of the PNG version and deleted the old PNG image. The SVG was optimized before being committed.

Bigger/clearer RStudio Bioconductor logo:
<img width="300" alt="Screen Shot 2022-04-08 at 2 10 34 PM" src="https://user-images.githubusercontent.com/97535660/162498258-6feb0d30-ae67-441f-ab4a-ca82b603784a.png">
<img width="600" alt="Screen Shot 2022-04-08 at 2 10 48 PM" src="https://user-images.githubusercontent.com/97535660/162498263-d7fda21f-3a1e-40b3-9d4f-dde35a122de2.png">
<img width="300" alt="Screen Shot 2022-04-08 at 2 13 12 PM" src="https://user-images.githubusercontent.com/97535660/162498416-2a800d36-cdcf-4ddd-85b2-c0be184505c3.png">

Before `Select an application` modal :
![Screen Shot 2022-03-29 at 10 03 02 AM](https://user-images.githubusercontent.com/97535660/160635478-60a1c13a-a84a-4a84-a3d0-97a48407e3ea.png)
After:
<img width="451" alt="Screen Shot 2022-04-08 at 2 15 23 PM" src="https://user-images.githubusercontent.com/97535660/162498695-46b1832d-3446-4810-876c-ee0c10a43ada.png">

cstrandw committed:
changed rstudio icon size

cstrandw committed:
changed rstudio icon size

cstrandw committed:
removed back arrow from modal after +Create

cstrandw committed:
changed rstudio logo size in +Create modal

cahrens commented:
@cstrandw please take a look at the test failures and get a deployment for the PR. Thanks!

jdcanas commented on src/components/AnalysisModal.js:
```diff
@@ -316,7 +316,7 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
           onDismiss()
         },
-        onPrevious: () => !!viewMode && resetView()
+        onPrevious: !!viewMode ? () => setViewMode(undefined) : undefined
```
Why do we not call reset view here anymore?

cahrens commented on src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js:
```diff
@@ -255,6 +255,10 @@ export const CloudEnvironmentModal = ({
 
+  const getToolIconHeight = toolLabel => Utils.switchCase(toolLabel,
+    [tools.RStudio.label, () => 30],
+    [Utils.DEFAULT, () => 20])
```
It would be nice to see the tool icon/tool icon height move into the tools data structure so that each tool is describes its desired behavior vs. having switch statements like `getToolIcon` and `getToolIconHeight`.

cahrens replied:
Actually, I wonder if this would be better handled by changing the RStudio logo itself to have the desired font size. It's sort of strange to change the height of the container to handle variable size of text within the logos.

(Also, now the heights/placement of "None" aren't completely consistent)

cstrandw replied:
Yeah, I agree that changing the logo font itself would be ideal, but how would you recommend going about changing the font size?

cahrens replied:
I would suggest asking UX to help you. They have edited svgs for me in the past.

kyuksel replied:
There's the #dsp-ui-ux channel which would be great to use for this purpose :)

cstrandw committed:
trying resizing images

cstrandw committed:
Merge branch 'dev' of github.com:DataBiosphere/terra-ui into cs-usability-nice-to-haves

cstrandw committed:
added new rstudio bio logo svg

cstrandw committed:
Merge branch 'dev' of github.com:DataBiosphere/terra-ui into cs-usability-nice-to-haves

cstrandw committed:
changed margins:

cstrandw commented:
> @cstrandw is this ready for review again?

Yes, it is. Thanks!

cahrens commented:
@cstrandw you don't need to check in the PNGs anymore, do you? And can you delete the PNG that is already checked in?

Also, did you optimize the SVG?

cstrandw committed:
deleted pngs

cstrandw commented:
> @cstrandw you don't need to check in the PNGs anymore, do you? And can you delete the PNG that is already checked in?
> 
> Also, did you optimize the SVG?

Deleted the PNGs, sorry I thought I had not pushed them but must have added them to the commit accidentally. And yes I did optimize the SVG, I just updated the PR description to include this.

cahrens commented on src/components/AnalysisModal.js:
```diff
@@ -315,7 +315,7 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
           onDismiss()
         },
-        onPrevious: () => !!viewMode && resetView()
+        onPrevious: !!viewMode ? () => resetView() : undefined
```
There is no functional difference here, correct?

cstrandw replied:
With the old version, onPrevious was never undefined, it was still a function object even if !!viewMode was false which allowed the arrow in the AnalysisModal TitleBar to show up when it shouldn't. With the second version, this is fixed as onPrevious can be undefined. This logic is consistent with other implementations of the TitleBar in the GalaxyModal as well.
